src/app/common/hooks/use-bitcoin-contracts.ts

```diff
@@ -215,6 +274,32 @@ export function useBitcoinContracts() {
     };
   }
 
+  function formatBitcoinContracts(
+    bitcoinContracts: Record<string, any>[]
+  ): BitcoinContractListItem[] {
+    return bitcoinContracts.map(bitcoinContract => {
+      return {
+        id: truncateMiddle(bitcoinContract.id),
```

Comment:
if this is just for presentation, wouldn't it be better to do this directly in the JSX where it's displayed, rather than in advance in the data structure?

Reply:
Yeah, removed this formatting function